bindings/netstandard/ElectionGuard/ElectionGuard.Encryption/ElGamalCiphertext.cs

```diff
@@ -73,6 +73,18 @@ public ElGamalCiphertext(ElementModP pad, ElementModP data)
             Handle = handle;
         }
 
+        public ElGamalCiphertext(ElGamalCiphertext other)
+        {
+            using (var pad = new ElementModP(other.Pad))
+            using (var data = new ElementModP(other.Data))
```

Comment:
Creating these new ElementModP were not really needed since the ElGamalCiphertext::make creates new ones with make_unique and in that constructor it already does a clone